### PR TITLE
getCookies bugfix

### PR DESCRIPTION
**Pull request by jeroenv**
getCookies method should check if the CookieManager instance isn't null before calling getCookie method.

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -20,8 +20,8 @@
 
 <plugin xmlns="http://apache.org/cordova/ns/plugins/1.0"
     xmlns:android="http://schemas.android.com/apk/res/android"
-    id="cordova-plugin-file-transfer"
-    version="1.3.1-dev">
+    id="checkroom-plugin-file-transfer"
```
Comment (daserge) [inline plugin.xml]: Looks like there are extra unrelated things in the PR, please remove them.

**Comment by daserge** [comment]
@jeroenv, thanks for your contribution!
Could you please file a [Jira bug](https://issues.apache.org/jira/browse/CB) with the issue details and environment?
The commits then should be squashed into a single one having CB-XXXX prefix.
Please also sign the ICLA so that your code could be pulled in: http://www.apache.org/dev/new-committers-guide.html#cla

**Comment by jeroenv** [comment]
@daserge added the Jira bug: https://issues.apache.org/jira/browse/CB-9879

**Comment by daserge** [comment]
Thanks, @jeroenv!
Please don't forget to sign the ICLA.

**Comment by daserge** [comment]
@jeroenv, I've tested `upload` with CrossWalk webview and have not found any issues.
Is there any additional details on how this issue could be reproduced?

**Comment by jeroenv** [comment]
@daserge You can reproduce the error on an Android device (4.1.2 or smaller) and use CrossWalk webview together with the file transfer plugin.

**Comment by daserge** [comment]
It does not reproduce on Android 4.0.3 device for me.

**Comment by jeroenv** [comment]
@daserge I forgot to mention that this bug only seems to happen with certain pictures you try to upload from Gallery/Dropbox/Google Drive. If you pick a picture from Photos or take a new picture and upload it everything works fine.

**Comment by daserge** [comment]
@jeroenv, I agree to @purplecabbage' reply to the dev-list:
> Post a picture that fails so we can verify. 

I have tested this with Google Drive - still no repro.
Which server do you use for upload? Does it reproduce with [cordova-vm](https://github.com/apache/cordova-plugin-file-transfer/blob/066acb33aff08e97f6bbb1bd1b9141e9e8238cd1/tests/tests.js#L42)?

**Comment by jeroenv** [comment]
@daserge All pictures selected from Dropbox fail when i use File Transfer plugin together with Crosswalk Webview.

**Comment by daserge** [comment]
@jeroenv, I could not reproduce this on 4.0.3 device with DropBox as well; the repro steps I tried were the following:
* Using camera plugin manual tests with cordova-plugin-test-framework and csp-include.js from mobilespec,
* sourceType: PhotoLibary/SavedPhotoAlbum,
* mediaType: AllMedia (to show apps including DropBox providing images),
* Get picture -> select DropBox,
    * Selected file path is in the following format: `file:///mnt/sdcard/Android/data/com.dropbox.android/files/u*******/scratch/file_name.jpg`

* Actions -> Upload image -> Upload complete

I've also tried to test on 4.1.2 emulator but the app does not run on it because of some CrossWalk issues: could not read AndroidManifest.xml on x86, and this error on arm:
`E/chromium( 2002): [ERROR:xwalk_platform_notification_service.cc(103)] Not implemented reached in virtual bool xwalk::XWalkPlatformNotificationService::GetDisplayedPersistentNotifications(content::BrowserContext*, std::set<std::basic_string<char, std::char_traits<char>, std::allocator<char> > >*)`.

**Comment by daserge** [comment]
Merged this to master:
https://git-wip-us.apache.org/repos/asf?p=cordova-plugin-file-transfer.git;a=commit;h=cdeec1c59fd8fcfa9159ad733081ac93ec060f5a

Github repo is out of sync now though.

**Pull request closed without merging**